### PR TITLE
flatpak: bump runtime to 40

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
## This PR fixes/implements the following **bugs/features**:
 

- Fixes #
- Fixes #

### CODE REVIEW

**Comment by Alecaddd** [comment]
@bilelmoussaoui the flatpak build is failing.
Elementary uses SCSS now to compile their stylesheet, so I think that's needed to build it from source.

**Comment by bilelmoussaoui** [comment]
> @bilelmoussaoui the flatpak build is failing.
> Elementary uses SCSS now to compile their stylesheet, so I think that's needed to build it from source.

I will have a look ^^

File: build-aux/flatpak/com.github.akiraux.akira.json [hunk 1]
```diff
@@ -1,56 +1,120 @@
...
+    "*.a",
+    "*.la"
+  ],
+  "finish-args": [
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--talk-name=org.gtk.vfs.*",
+    "--filesystem=xdg-run/gvfs",
+    "--filesystem=xdg-run/gvfsd"
```
Comment (bilelmoussaoui) [inline build-aux/flatpak/com.github.akiraux.akira.json:24]: Those were always needed to be able to open file from another hard drive, remote mounts...

**Comment by Alecaddd** [comment]
Awesome, thank you so much!

**Review by Alecaddd — APPROVED** [review]

**Pull request merged**